archive-resource.php

```diff
@@ -170,52 +151,68 @@
 
 			</div>
 
-			<a href="#" class="resource__close" v-on:click.prevent="open_resource = null"><svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-close"></use></svg></a>
+			<a href="#" class="resource__close" v-on:click.prevent="closeResource()"><svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-close"></use></svg></a>
 
 		</div> <!-- / .resource -->
 
-		<template id="resource-template">
+		<script type="x/templates" id="resource-template">
 
-			<div class="post-object--resource__media">
+			<div class="card__header / card--content-spine" data-content-type="resource">
 
-				<div class="post-object--resource__icon">
+				<div class="content-spine__icon">
 					<svg><use xlink:href="#icon-resource"></svg>
 				</div>
 
-				<div class="post-object--resource__type">
-					<span v-for="type in resource.taxonomies['resource-type']">{{{ type }}}</span>
-				</div>
+				<span class="content-spine__title">{{{ type }}}</span>
+
+				<span class="content-spine__meta">{{{ resource.custom.year }}}</span>
 
 			</div>
 
-			<div class="post-object__content / media__body">
+			<div class="card__content">
 
-				<p class="post-object__meta">
-					<span v-if="resource.fields.organisation !== ''"><?php _e('By', 'ocp'); ?> {{ resource.fields.organisation }}</span>
-					<time>{{ resource.custom.year }}</time>
-				</p>
+				<div class="card__title">
 
-				<span><h4>{{{ resource.title }}}</h4></span>
+					<h6 class="card__heading">
+						<a class="card__link" href="{{ resource.link }}">{{{ resource.title }}}</a>
+					</h6>
+
+				</div>
+
+				<p class="card__meta" v-if="resource.fields.organisation !== ''">
+					<span class="card__author"><?php _e('By', 'ocp'); ?> {{ resource.fields.organisation }}</span>
+				</p>
 
 			</div>
 
-		</template>
+		</script>
 
 		<script>
```

Comment:
That's a lot of blocking inline JavaScript...

This will load in before any scripts enqueued by WordPress, could be dangerous.